pkg/txm/txm.go

```diff
@@ -199,12 +199,15 @@ func (t *Txm) broadcastWithRetry(ctx context.Context, tx *Tx, msg *wallet.Messag
 	var receivedMessage *tracetracking.ReceivedMessage
 	var err error
 
+	// TODO: fix shadowed error
```

Comment:
I see this logic is broken now, but we could simplify by doing the client get before the loop, so we are only retrying the method call. Bit of a tradeoff, but seems valid.